source/content/guides/multisite/07-search-replace.md

```diff
@@ -0,0 +1,53 @@
+---
+title: WordPress Multisite
+subtitle: Search & Replace
+description: Additional handing required for running search & replace on WordPress Multisites.
+type: guide
+getfeedbackform: default
+contenttype: [guide]
+innav: [false]
+categories: [cms, troubleshooting]
+cms: [wordpress]
+audience: [development]
+product: [--]
+integration: [--]
+tags: [multisite]
+permalink: docs/guides/multisite/search-replace/
+editpath: multisite/07-search-replace.md
+---
+
+<Alert title="Early Access" type="info" icon="leaf">
+
+WordPress Multisite (WPMS) search & replace is available for [Early Access](/oss-support-levels#early-access) participants. Features for WPMS search & replace are in active development. Pantheon's development team is rolling out new functionality often while this product is in Early Access. Visit [#wordpress in our community Slack](https://pantheon-community.slack.com/archives/CT8MC5Y0K) (you can sign up for the [Pantheon Slack channel here](https://slackin.pantheon.io/) if you don't already have an account) to learn how you can enroll in our Early Access program. Please review Pantheon's [Software Evaluation Licensing Terms](https://legal.pantheon.io/#contract-hkqlbwpxo) for more information about access to our software.
```

Comment:
```suggestion
WordPress Multisite (WPMS) search & replace is available for [Early Access](/oss-support-levels#early-access) participants. Features for platform search-replace are in active development. Pantheon's development team is rolling out new functionality often while this product is in Early Access. Visit [#wordpress in our community Slack](https://pantheon-community.slack.com/archives/CT8MC5Y0K) (you can sign up for the [Pantheon Slack channel here](https://slackin.pantheon.io/) if you don't already have an account) to learn how you can enroll in our Early Access program. Please review Pantheon's [Software Evaluation Licensing Terms](https://legal.pantheon.io/#contract-hkqlbwpxo) for more information about access to our software.
```

This is referred to as `platform search-replace` further down in this document - we should decide on `WPMS search-replace` or `platform search-replace` and be consistent throughout the doc.

Reply:
re: WPMS search & replace vs platform search & replace— trying to differentiate as this behaves somewhat differently than _any s&r_ on the platform, and you can't *opt-in/out* of s&r on a non-WPMS site so that seemed confusing.